plugins/service/aperture-plugin-fluxninja/otel/provide.go

```diff
@@ -128,9 +129,13 @@ func addFNToPipeline(
 
 func addMetricsSlowPipeline(baseConfig, config *otelcollector.OTELConfig) {
 	addFluxninjaPrometheusReceiver(baseConfig, config)
+	addFluxninjaKubeletStatsReceiver(baseConfig, config)
```

Comment:
Why is this in the fluxninja plugin? It's better to pass this config explicitly via [user visible config](https://github.com/fluxninja/aperture/issues/1123) so that the user can enable the precise metrics they need. We would enable only a select few metrics by default

Reply:
The part in fluxninja plugin is not defining this pipeline from scratch, but it is duplicating the original pipeline with two changes:
1. Increased scrape interval from `1s` to `10s` as we don't need such granularity in cloud metrics
2. Added fluxninja exporter which is writing metrics to the cloud

I can wait for the user defined configs to be completed and modify this PR to use this.